src/views/pair.py

```diff
@@ -6,9 +6,9 @@
 from uuid import uuid4
 from requests import HTTPError
 import threading, time
-from mycroft_bus_client import Message
-from mycroft.api import is_paired
-from mycroft.identity import IdentityManager
+from ovos_bus_client import Message
+from ovos_backend_client import is_paired
+from ovos_backend_client.identity import IdentityManager
```

Comment:
it seems this file can be deprecated, by default now ovos runs offline and pairing is not necessary

if users decide to use a personal-backend then the pairing is done manually by the user  (creating identity2.json provided by backend + enabling it in mycroft.conf)